deploy/storageos-operators.configmap.yaml

```diff
@@ -463,7 +463,7 @@ data:
 
     - packageName: storageos
       channels:
-      - name: alpha
+      - name: stable
         currentCSV: storageosoperator.0.0.0
```

Comment:
Is `0.0.0` correct?

Reply:
yes, this is used in the e2e test. We need not increment it every time.